PyBambooHR/PyBambooHR.py

```diff
@@ -22,6 +22,13 @@
 except NameError:
     # unicode is undefined: We are running Python 3
     unicode = str
+
+    # basestring is undefined: We are running Python 3
```

Comment:
this branch only executes when `unicode` is not defined, i.e. when we're already running under python 3. shouldn't 
this all be un-indented?